messaging-activemq/src/test/resources/org/wildfly/extension/messaging/activemq/subsystem_12_0_transform.xml

```diff
@@ -72,6 +72,11 @@
             network-check-ping6-command="ping6 -c 1 %2$s"
             network-check-timeout="1000"/>
 
+            <critical-analyzer
```

Comment:
Assuming this is for the 'working' configuration for transformation, I am curious why it allows transformation when these are defined. I would have thought they should get rejected (unless these values are what is implied in the old server version)?

Reply:
Those are the default values and thus are ignored